textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/queryrewriter/QueryRewriter.java

```diff
@@ -0,0 +1,77 @@
+package edu.uci.ics.textdb.dataflow.queryrewriter;
+
+import edu.uci.ics.textdb.api.common.*;
+import edu.uci.ics.textdb.api.dataflow.IOperator;
+import edu.uci.ics.textdb.common.field.DataTuple;
+import edu.uci.ics.textdb.common.field.ListField;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by kishorenarendran on 25/04/16.
+ * QueryRewriter is an operator that converts a query search string which
+ * has faults in space placement to a list of meaningful search queries
+ * QueryRewriter internally uses the FuzzyTokenizer class to fulfill
```

Comment:
Remove the sentence "QueryRewriter internally uses the FuzzyTokenizer class to fulfill its functionality" since it's related to the coming PR of implementation.